source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -0,0 +1,153 @@
+
+   With the default parity of ``EC:4``, the deployment can tolerate the loss of 4 drives per erasure set and still serve read operations.
+
+Write quorum depends on the configured parity and the size of the erasure set.
+   If parity is less than 1/2 the number of erasure set drives, write quorum equals parity and functions similarly to read quorum.
+   MinIO automatically "upgrades" the parity of objects written to a degraded erasure set to ensure that object can meet the same SLA as objects in healthy erasure sets.
+
+   .. figure:: /images/availability/availability-erasure-sharding-degraded-write.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of degraded erasure set, where two drives have failed
+
+      This node has two failed drives.
+      MinIO writes the object with an upgraded parity of ``EC:6`` to ensure this object meets the same SLA as other objects.
+
+   With the default parity of ``EC:4``, the deployment can tolerate the loss of 4 drives per erasure set and still serve write operations.
+   MinIO can perform "parity upgrade" up to 1/2 the drives in the erasure set.
+
+If parity equals 1/2 the number of erasure set drives, write quorum equals parity + 1 to avoid data inconsistency due to "split brain" scenarios.
+   For example, if exactly half the drives in the erasure set become isolated due to a network fault, MinIO would consider quorum lost as it cannot establish a N+1 group of drives for the write operation.
+
+   .. figure:: /images/availability/availability-erasure-sharding-split-brain.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of erasure set where half the drives have failed
+
+      This node has 50% drive failure.
+      If parity is ``EC:8``, this erasure set cannot meet write quorum and MinIO rejects write operations to that set.
+      Since the erasure set still maintains read quorum, read operations to existing objects can still succeed.
+
+An erasure set which loses more drives than the configured parity has suffered data loss. 
+   For maximum parity configurations, the erasure set goes into "read only" mode if drive loss equals parity.
+   For the maximum erasure set size of 16 and maximum parity of 8, this would require the loss of 9 drives for data loss to occur.
+
+   .. figure:: /images/availability/availability-erasure-sharding-degraded-set.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of completely degraded erasure set
+
+      This erasure set has lost more drives than the configured parity of ``EC:4`` and has therefore lost both read and write quorum.
+      MinIO cannot recover any data stored on this erasure set.
+
+MinIO further mitigates the risk of erasure set failure by "striping" erasure set drives across each node in the pool.
+   MinIO automatically calculates the optimal erasure set size based on the number of nodes and drives.
```

Comment:
Maybe mention here or somewhere that erasure sets contain no more than 16 drives.